### PR TITLE
Add helm charts as deployment option

### PR DESCRIPTION
**Pull request by sleyer**

### CODE REVIEW

File: deployment/helm/image-metadata-orchestrator-base/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+{{- define "image-metadata-orchestrator-base.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "image-metadata-orchestrator-base.fullname" -}}
```
Comment (wurstbrot) [inline deployment/helm/image-metadata-orchestrator-base/templates/_helpers.tpl:13]: We have the "image metadata collector".
This app name could be "cluster-image-scanner-orchestrator"

File: deployment/helm/image-metadata-orchestrator-base/templates/depcheck.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (wurstbrot) [inline deployment/helm/image-metadata-orchestrator-base/templates/depcheck.yml:1]: this file can be removed

File: deployment/helm/image-metadata-orchestrator-base/templates/_helpers.tpl (hunk 1 above)
Comment (sleyer) [reply deployment/helm/image-metadata-orchestrator-base/templates/_helpers.tpl:13]: We agreed to keep the name as is.

File: deployment/helm/image-metadata-orchestrator-base/templates/depcheck.yml (hunk 2 above)
Comment (sleyer) [reply deployment/helm/image-metadata-orchestrator-base/templates/depcheck.yml]: Done

**Comment by wurstbrot** [comment]
TODO: rename "image-metadata-orchestrator" to "cluster-image-scanner-orchestrator"

**Comment by wurstbrot** [comment]
https://github.com/SDA-SE/terraform-nimbuskube/blob/xxx-1810_clusterscanner-orchestrator/modules/clusterscanner-orchestrator/variables.tf#L111 -> der default NS sollte "clusterscanner" sein. argowf verwenden wir bei der SDA SE, weil wir es nicht "richtig" kurzfristig hinbekommen haben

**Comment by wurstbrot** [comment]
SMTP vars are missing, I will take a look

**Comment by wurstbrot** [comment]
@sleyer  why do you have labes in orchestrator templates and not in orchestrator-base templates?

```
  labels:
    {{- include "cluster-image-scanner-orchestrator.labels" .  | nindent 4 }}
 ```

**Review by ChristianSpiess — APPROVED** [review]

**Comment by wurstbrot** [comment]
close, siehe https://github.com/SDA-SE/cluster-image-scanner/pull/121

**Pull request merged**